pages/_app.tsx

```diff
@@ -45,7 +43,6 @@ export default function Layout({
         <title>BitcoinBeach Lightning Node</title>
       </Head>
       <GraphQLProvider>
```

Comment:
@UncleSamtoshi @samerbuna Typescript throws a warning here `GraphQLProvider' cannot be used as a JSX component` Any ideas on how to remove this except using `@ts-ignore`

Reply:
Resolved by updating react-types